src/SFA.DAS.Assessor.Functions/Functions/Print/PrintFunctionFlow.cs

```diff
@@ -16,7 +16,7 @@ public PrintFunctionFlow(IPrintProcessCommand command)
         }
 
         [FunctionName("PrintFunctionFlow")]
-        public async Task Run([TimerTrigger("0 */15 * * * *", RunOnStartup = true)]TimerInfo myTimer, ILogger log)
+        public async Task Run([TimerTrigger("%PrintFunctionFlowFunctionSchedule%", RunOnStartup = true)]TimerInfo myTimer, ILogger log)
```

Comment:
Double use of the word function in the app setting name?

Reply:
Fixed